test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -724,16 +730,17 @@ public void requestResetPassword() {
         
         participantService.requestResetPassword(STUDY, ID);
         
-        verify(accountDao).requestResetPassword(eq(STUDY), emailCaptor.capture());
+        verify(accountDao).requestResetPassword(accountIdCaptor.capture());
```

Comment:
Nitpick: Would eq(ACCOUNT_ID) be better than use a capture?

Reply:
Yes, it's better, changed.